test/functional/locations_controller_test.rb

```diff
@@ -7,6 +7,7 @@ class LocationsControllerTest < ActionController::TestCase
   end
 
   test "should get edit" do
+    Permission.create :name => 'view_provisioning_templates', :resource_type => 'Location'
```

Comment:
Is this the right place to do it?  I feel it should probably be in fixtures for now, as 98% of the rest already are... perhaps another day we can refactor them out into factories or something so they're only created when required.